app/views/grantees/show.html.erb

```diff
@@ -20,7 +28,7 @@
     </div>
     <div class="grid-col-8">
       <ul class="usa-process-list">
-        <% (1..5).each do |index| %>
+        <% (1..@grantee.complaints_per_fy).each do |index| %>
```

Comment:
[question] I think at the moment we are expecting that this is a number, right?  Like, that a grantee had 87 complaints for the past fiscal year.  There is a `relationship` structure in the request where we're hoping HSES will respond with the actual complaint data. Perhaps this section would be getter served grabbing that info?

Reply:
@jduss4 yes, i think so, but when we incorporate the complaint data into this. this is just the grantee data, so it felt out of scope with this issue. i wanted to do this ticket first so we would have the fake grantee endpoint first so we could connect the fake data in a way that mirrored what might happen with the real data

Reply:
Hmm okay, I can live with that although I think I'd rather just display one or two placeholders rather than a semi-arbitrary set of numbers.